app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -853,6 +915,7 @@ def rbac_group_get_form_vars
       @edit[:new][:description] = params[:ldap_groups_user] if params[:ldap_groups_user]
       @edit[:new][:description] = params[:description]      if params[:description]
       @edit[:new][:role]        = params[:group_role]       if params[:group_role]
+      @edit[:new][:tenant]      = params[:group_tenant]       if params[:group_tenant]
```

Comment:
why `:group_tenant` instead of `tenant`?  How are `group` and `tenant` related here?